api/Services/OfferServices/OfferService.cs

```diff
@@ -203,12 +205,17 @@ public async Task<string> AddOffer(OfferCreateDto offerDto, string userEmail)
 
             await _offerRepository.AddOffer(offer);
 
+            if (!await _fileService.AddOfferImages(offerDto.Images, offer.Id))
+            {
+                return "Error while creating offer";   
+            }
+
```

Comment:
To wszystko? Jak ktoś poda za dużo zdjęć to mu nawet nie powiemy tylko dostanie że Error while creating offer? Zastanowiłbym się nad tym żeby te odpowiedzi były bardziej semantyczne